compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -17,8 +17,10 @@ use rustc_ast::ptr::P;
 use rustc_ast::token::{self, DelimToken, Token, TokenKind};
 use rustc_ast::tokenstream::{self, DelimSpan, TokenStream, TokenTree, TreeAndJoint};
 use rustc_ast::DUMMY_NODE_ID;
-use rustc_ast::{self as ast, AttrStyle, AttrVec, Const, CrateSugar, Extern, Unsafe};
-use rustc_ast::{Async, MacArgs, MacDelimiter, Mutability, StrLit, Visibility, VisibilityKind};
+use rustc_ast::{self as ast, AnonConst, AttrStyle, AttrVec, Const, CrateSugar, Extern, Unsafe};
+use rustc_ast::{
+    Async, ExprKind, MacArgs, MacDelimiter, Mutability, StrLit, Visibility, VisibilityKind,
+};
```

Comment:
Nit: could you avoid multi-line imports?

Reply:
Just in case, what do you exactly mean here?. I don't like in the way things were before modifying the code but I've tried lo leave things as close as similar as they were. I'm happy to fix all this as a drive by but I'm not sure if you want me to join those two last imports into one use, or if you want to avoid { } or even merge all the five previous use there.
I think I'd merge the last three lines that are use statements in the root path and do not bother with rustc_ast::tokenstream and the rest.